src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,13 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length === 0 ? (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    ) : (
+      comments.map(comment => (
+        <CommentInfo comment={comment} key={comment.id} />
```

Comment:
The `key` should be placed on the root tag inside the iteration. In this case, it should be on the `<CommentInfo>` component when it's being used inside the `.map` call (CHECKLIST: 1).